silkworm/rpc/http/connection.cpp

```diff
@@ -190,10 +192,11 @@ Task<void> Connection::handle_actual_request(const boost::beast::http::request<b
     vary_ = req[boost::beast::http::field::vary];
     origin_ = req[boost::beast::http::field::origin];
     method_ = req.method();
+    auto encoding = req[boost::beast::http::field::accept_encoding];
```

Comment:
`encoding` should be declared `const` (next PR is OK for this and all following suggestions)